### DISCUSSION TITLE
【Notion图片】notion上的图片加载很慢

### DISCUSSION THREAD

**Discussion by hehan-wang** [tangly1024/NotionNext#2102] [Ideas]
**描述bug**
下面这个notion上的图片国内加载很慢，请教下有好的解决办法么？
https://www.notion.so/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2F5fdde3cf-8f5c-4292-84e6-0bb16a36f7d7%2F0c324d7d-714f-4783-a37f-441a5d417dcf%2FUntitled.png?table=block&id=7ad0cfaf-e651-42c6-ba8b-5f16961fd120

**环境**

- 操作系统: macOS
- 浏览器 chrome
- NotionNext版本 4.0.14
- 主题 heo

**Comment by hehan-wang** [comment]
我自己可以使用翻墙加速，但是不利于分享给其他小白用户

**Comment by happyzhangyyds** [comment]
您可以试着使用picgo+自定义图床进行加速，例如杜老师图床

**Comment by happyzhangyyds** [comment]
同时建议使用图片压缩工具对图像先行进行压缩，例如Mac上可以使用 “ImageOptim”，同时建议采取合适的图片格式，例如jpg，jpeg，也可以看看这篇文章：https://matrixcore.top/article/picturespeed

**Comment by 821869798** [comment]
同问，如果要手动上传到别的cdn上，写笔记的时候会很麻烦，还需要上传，拿连接。
